### PR TITLE
Illia rudniev/feat/token auth kyb

### LINKED ISSUE #1047: KYB - Token auth
- remove customer secrets keys from KYB app
- update collection flow controller to fetch all relevant KYB data by token
- implement token auth guard for collection flow controller
- implement token scope decorator to include all relevant information about session (runtime id,definition id, project scopes)

### PR DESCRIPTION
**Pull request by chesterkmr**
### Description
- Added support of token in KYB app
- Added token guards & decorators
- Updated collection flow controller to work with tokens
- Updated mapping of documents on WF update

### Related issues
 * Provide a link to each related issue.

### Breaking changes
 * Describe the breaking changes that this pull request introduces.

### How these changes were tested
 * Describe the tests that you ran to verify your changes, including devices, operating systems, browsers and versions.

### Examples and references
 * Links, screenshots, and other resources related to this change.

### Checklist
- [] I have read the [contribution guidelines](CONTRIBUTING.md) of this project
- [] I have read the [style guidelines](STYLE_GUIDE.md) of this project
- [] I have performed a self-review of my own code
- [] I have commented my code, particularly in hard-to-understand areas
- [] I have made corresponding changes to the documentation
- [] My changes generate no new warnings and errors
- [] New and existing tests pass locally with my changes

### CODE REVIEW

File: examples/kyb-app/src/hocs/withTokenProtected/index.ts
Comment (MatanYadaev) [inline examples/kyb-app/src/hocs/withTokenProtected/index.ts:1]: This file is empty. Remove or fix it?

File: examples/kyb-app/src/hocs/withTokenProtected/withTokenProtected.tsx [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+export function withTokenProtected<TComponentProps>(
+  Component: React.ComponentType<TComponentProps>,
+) {
+  function Wrapper(props: TComponentProps) {
+    const accessToken = useMemo(() => getAccessToken(), []);
+
+    if (!accessToken) return <AppNavigate to="restricted" />;
+
+    console.log({ accessToken });
```
Comment (MatanYadaev) [inline examples/kyb-app/src/hocs/withTokenProtected/withTokenProtected.tsx]: Leftover :)

File: examples/kyb-app/src/hooks/useSessionQuery/useSessionQuery.ts [hunk 2]
```diff
@@ -1,14 +1,13 @@
...
 
 export const useSessionQuery = () => {
   const { data: user = null, isFetching } = useQuery({
-    ...collectionFlowQuerykeys.getSession({ email: localStorage.getItem(TOKEN_KEY) }),
-    enabled: Boolean(localStorage.getItem(TOKEN_KEY)),
+    ...collectionFlowQuerykeys.getSession(),
     staleTime: Infinity,
   });
 
+  console.log('user', user);
```
Comment (MatanYadaev) [inline examples/kyb-app/src/hooks/useSessionQuery/useSessionQuery.ts]: Leftover :)

File: examples/kyb-app/src/hooks/useSignin/useSignin.ts [hunk 3]
```diff
@@ -9,15 +9,13 @@ export const useSignin = () => {
 
   const signin = useCallback(async ({ email }: SigninCredentials) => {
     setSigningIn(true);
-    await authorizeUser({ email });
-
-    localStorage.setItem(TOKEN_KEY, email);
+    // await authorizeUser({ email });
```
Comment (MatanYadaev) [inline examples/kyb-app/src/hooks/useSignin/useSignin.ts]: Let's remove dead code like this (also in line 1 and 18).

File: examples/kyb-app/src/pages/SignIn/SignIn.tsx [hunk 4]
```diff
@@ -60,4 +59,5 @@ export const SignIn = () => {
       </div>
     </SigninLayout>
   );
+  return null;
```
Comment (MatanYadaev) [inline examples/kyb-app/src/pages/SignIn/SignIn.tsx]: Redundant?

File: services/workflows-service/scripts/workflows/parent-kyb-workflow.ts [hunk 5]
```diff
@@ -78,7 +78,7 @@ export const kybParentDynamicExample = {
                 countryOfIncorporation: entity.data.countryOfIncorporation,
                 companyNumber: entity.data.registrationNumber,
                 state: entity.data.additionalInfo.company.state
-                vendor: 'open-corporates'
+                vendor: 'veriff'
```
Comment (MatanYadaev) [inline services/workflows-service/scripts/workflows/parent-kyb-workflow.ts]: Why? It's a request to OpenCorporates, not to Veriff

Comment (chesterkmr) [reply services/workflows-service/scripts/workflows/parent-kyb-workflow.ts]: @MatanYadaev  Post merge,havent touched it

File: services/workflows-service/src/workflow/workflow.service.ts [hunk 6]
```diff
@@ -1581,6 +1583,7 @@ export class WorkflowService {
       {},
       projectIds,
     );
+    console.log('runtime data', workflowRuntimeData);
```
Comment (MatanYadaev) [inline services/workflows-service/src/workflow/workflow.service.ts]: Leftover

**Review by MatanYadaev — APPROVED** [review]
GREAT WORK MAN!🔥

File: services/workflows-service/scripts/workflows/parent-kyb-workflow.ts (hunk 5 above)
Comment (MatanYadaev) [inline services/workflows-service/scripts/workflows/parent-kyb-workflow.ts]:
```suggestion
                vendor: 'open-corporates'
```

**Review by MatanYadaev — APPROVED** [review]
Great job. If you QAed it and it works fine, let's merge it :)

**Comment by Blokh** [comment]
how do i you run it locally using token and kyb app?

**Review by tomer-shvadron — APPROVED** [review]

**Pull request merged**